docs/tutorials/enterprise_beginner/1_get_a_starter_website.md

```diff
@@ -51,6 +51,11 @@ Edit it to remove the Image Field that has a Content Relation (ezobjectrelation)
 
 ![New image Field in the Article Content Type](img/enterprise_tut_image_in_article_ct.png)
 
+!!! note "Alternative text requirement"
+
+    In some cases you might want to require an alternative text for the image. 
+    To do this, select the "Alternative text is required" checkbox.
+
```

Comment:
Selecting alternative text as mandatory should be our recommendation here. It's usually (and by default) used as `alt` attribute value and it's its value must not be empty. 

Ref. https://html.spec.whatwg.org/multipage/images.html#general-guidelines